examples/powertools-examples-core/kotlin/gradle/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id("io.freefair.aspectj.post-compile-weaving") version "6.6.3"
+    kotlin("jvm") version "1.9.10"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("com.amazonaws:aws-lambda-java-core:1.2.2")
+    implementation("com.fasterxml.jackson.core:jackson-annotations:2.13.2")
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.13.2.2")
+    implementation("com.amazonaws:aws-lambda-java-events:3.11.0")
+    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.13.2")
+    aspect("software.amazon.lambda:powertools-tracing:1.17.0")
```

Comment:
These still need to be set to `1.18.0-SNAPSHOT` and I guess pulled into a param or so